tests/e2e/framework/kubernetes.go

```diff
@@ -0,0 +1,248 @@
+package framework
+
+import (
+	"bufio"
+	"flag"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+
+	"github.com/golang/glog"
+	"istio.io/istio/tests/e2e/util"
+)
+
+const (
+	VERSION_FILE = "istio.VERSION"
+	YAML_SUFFIX  = ".yaml"
+)
+
+var (
+	kube               KubeInfo
```

Comment:
This seems redundant to the flag. I don't understand why they are defined twice.